man/fi_poll.3.md

```diff
@@ -181,8 +181,9 @@ struct fi_wait_attr {
 : Indicates that the wait set should use file descriptor(s) as its wait
   mechanism. It may not always be possible for a wait set to be implemented
   using a single underlying file descriptor, but all wait objects will be file
-  descriptors. File descriptor wait objects must be usable in select, poll,
-  and epoll routines. However, a provider may signal an FD wait object by
+  descriptors. File descriptor wait objects must be usable in the
+  POSIX select(2), poll(2), and epoll(7) routines (if
+  available). However, a provider may signal an FD wait object by
   marking it as readable, writable, or with an error.
```

Comment:
There is a missed update here.  The fd may be signaled by marking it as readable or with an error.  We agreed to remove writable from this, as an fd may always be writable, making any attempt to wait on it useless.  Can you add this change?